### DISCUSSION TITLE
Bevy Logo & Identity Iteration

### DISCUSSION THREAD

**Discussion by thefuntastic** [bevyengine/bevy#9527] [Ideas]
According to @cart via the discord:
> im always open to iterative improvements to the logo. the general concept should remain the same, but if somebody can make something i like better i wont say no. 

This issue captures infomration about requirements for new logo.

**Comment by thefuntastic** [comment]
First draft sketch suggestion, will pursue further refinement and vectorisation if there is appetite for it
![image](https://user-images.githubusercontent.com/296286/90389508-9c8b4f80-e081-11ea-984b-eb991ecb62c4.png)

**Comment by aevyrie** [comment]
Something that was discussed earlier in discord was defining the branding before doing any UX/website redesign work. Part of that was settling on color choices and brand aesthetic. That might be something that we could work on here at the same time as the logo design itself, instead of having an issue for every branding subtask.

**Comment by awulkan** [comment]
I think I like the original logo a bit more in some ways. For example, I like that the first bird is flying with its chest out. It makes it look kinda like a bold super hero pose.

![pose](https://i.imgur.com/n0ynjjZ.png)

**Comment by thefuntastic** [comment]
@aevyrie that's a good point. I hesitated whether I should try make the issue broader scope to include branding. Slightly edited the title to reflect that. 

Agree that typeface and color palettes are important considerations too that will need to be discussed as part of a whole

**Comment by cart** [comment]
I think this is a case of "I'll know the next logo when I see it".

In general logo design requires a lot of experimentation to find what feels "right". And side-by-side comparison helps you "rank" logos in a way that is really hard when done individually.

To lift the veil on the process I used for the current logo, here is my "bevy logo scratch" file. Note that many of the designs don't work, but this style of brainstorming really gets the creative juices flowing.


![bevy_logo_scratch](https://user-images.githubusercontent.com/2694663/90442064-de6ec280-e08e-11ea-9fb9-2753d9fcced9.png)


svg: 
[bevy_logo_scratch.zip](https://github.com/bevyengine/bevy/files/5086682/bevy_logo_scratch.zip)

**Comment by aevyrie** [comment]
I was wondering if you tried tessellating them to make an Escher-like design; looks like some of the concepts in the top right were in that vein. The current logo/type has a playful feel, is that something that's important to retain if it does change?

I like the dynamic pose of the current design, like @awulkan pointed out.

**Comment by cart** [comment]
I highly value the "playful feel", but if the logo finds "character" with a different tone, I'm down. I'd also like to point out that I'm currently decently satisfied with the current design, so I won't be expending any effort constructing new designs. But if other people want to continue experimenting thats totally fine. I'll be watching this thread.

**Comment by mvit** [comment]
![Artboard Copy 2](https://user-images.githubusercontent.com/128863/90582004-ae144a80-e19a-11ea-9a8e-5a08cf3f7512.png)
![Artboard Copy](https://user-images.githubusercontent.com/128863/90582006-af457780-e19a-11ea-94ff-10168260fbcf.png)
![Artboard](https://user-images.githubusercontent.com/128863/90582008-af457780-e19a-11ea-8a28-518682e59fe3.png)

Experimented around a bit, ended up with these possible designs.
My focuses were simple, hint at a "b" shape, make the birds look up right always and have the flock in perspective as opposed to just downward rotation.

Am considering getting rid of the tails of the background birds though, as they just seem to clutter up the overall silhouette.

**Comment by zicklag** [comment]
Not to discourage other attempts, but I am very happy with the current logo. I think it's pretty much perfect. :slightly_smiling_face:

**Comment by thefuntastic** [comment]
There is the question floating aroud of "Do we even need a new logo (and brand indentity)"?

This will be subjective: to me the current logo looks like "programmer" art (no offense to cart). I think that is totally fine whilst Bevy's primary audience is, well programmers. The [Rust Logo](https://upload.wikimedia.org/wikipedia/commons/thumb/d/d5/Rust_programming_language_black_logo.svg/1200px-Rust_programming_language_black_logo.svg.png), [GNU logo](https://upload.wikimedia.org/wikipedia/en/thumb/2/22/Heckert_GNU_white.svg/1200px-Heckert_GNU_white.svg.png) strike a similar utilitarian asthetic and do just fine. 

However game dev is a broad (and colourful) church. People do use aethestics as a shorthand for quailty and trust. What conclusions would artists and game designers draw when Bevy eventually has an editor and they're looking for the next thing? I mean I assume so, but is that an audience we'd ever want to appeal to?

**Comment by tamasfe** [comment]
I personally quite like the current logo, I was just a bit bothered by the jagged lines visible when the logo is big enough (like in the website header) and the mix of straight lines and curves. So I removed the excessive nodes, and made it a bit curvier (though it could be worked on further):

![bevy_logo](https://user-images.githubusercontent.com/25967296/90806177-f0659500-e31c-11ea-874f-6fcd5588a43c.png)

[svgs](https://github.com/bevyengine/bevy/files/5104654/bevy.zip)

Whatever logo you end up with, I'd avoid detailed shapes or stacking stuff, as the logo gets smaller it might become a blurry mess. I think the current one is just fine for both small and large representations.

**Comment by awulkan** [comment]
@tamasfe I really like those small changes! The curvier look also fits the font choice better imo.

**Comment by regexident** [comment]
![Comparison](https://user-images.githubusercontent.com/138017/90842202-f9775600-e35e-11ea-9991-af822a2a9003.png)

**Comment by cart** [comment]
@tamasfe I like the idea of simplified curves, but I think I would prefer some sort of interpolation between the current logo's proportions and yours. Edit: I also think i prefer even spacing for the "gaps". I think @regexident is probably on the right track here.

@regexident I really like the simplified curves and rounder "corners". the larger gap also makes it more legible at smaller sizes. I'm not super sold on the new head design, but I do think it fits with the rest of the other changes.

**Comment by kettle11** [comment]
![Zoom](https://user-images.githubusercontent.com/4565191/90946980-0b044080-e3e7-11ea-8115-07065796b986.png)

Zoooooom

And here's the Figma doc for reference (along with with other experiments): https://www.figma.com/file/hUAlpcAdEawUTOOqslOToc/Bevy-Logo-Experiments?node-id=40%3A24

**Comment by GabCampbell** [comment]
What i would like about zoom is if the tip of the wing would go to the tip of the tail and if the line was less thicc. Also I think zoom and traditional bevy can coexist. I can also imagine a zoom animation going from left to right where at first its just the white bird and as it moves right the other ones appear

**Comment by GabCampbell** [comment]
@cart i agree with that even spacing with tamasfe's version of bevy is actually a great starting point (almost final)

**Comment by regexident** [comment]
@cart glad you like the overall direction. I'm not entirely sold on the new head either. I had some troubles adapting the existing head without having it look like it was bolted on. Hence the change. I agree though in that it still needs some more iterations and work. It looks a bit like a penguin dropping into the water now, rather than the Darwin Finken I was aiming for. 😄

I'm away from my computer right now and for a couple more days, so screenshots of the logo's construction or further iterations will unfortunately have to wait until beginning of next month.

The overall idea was to construct the logo from geometric shapes (i.e. circles), so that the individual curves share their radii. If you align a circle to the upper wing's upper curve and move it over some of the other larger curves you'll see that they share a single radius. Further more I didn't use arbitrary circles, but a fixed set of circles scaled in intervals of the golden ratio. This type of construction tends to result in a clean and consistent look and also helps in ensuring curvature continuity (i.e. preventing bumps).

These random Dribbble designs: [#1](https://dribbble.com/shots/12048231-Petocation-Logo-Design) and [#2](https://dribbble.com/shots/12216105-Home-Spot-Logo-Grid), should give you an idea of what I'm talking about.

Happy to answer questions in the meanwhile.

**Comment by tamasfe** [comment]
@regexident Mapping most things to ellipses is definitely a way to go,

I would make some changes to your version:
- Retain only a single node at the tail/wingtips to make them pointier (the same edit I did to the original)
- Rework the head, or just the straight line instead of a beak, it makes it look chopped off
- Place the shadows with an angle instead of being fully horizontal while keeping the even spacing, I think that makes the original more interesting (to me at least)

I could also have a stab at later it if you upload the svgs.

**Comment by regexident** [comment]
@tamasfe I am gonna be on phone-only for the next couple days, without access to the SVG, unfortunately. :/

### Wings

I prefer the wings without pointy corners.
Rounded corners have a friendlier feel and also work better in small sizes, as tapered/thin details are often lost at small sizes. They also work nice with the overall rounded look of the logo. It's all curves. (Another reason to change the head, which features the only straight line right now.)

### Beak

Agreed. The beak will definitely see some more iterations once I'm back to it by the end of next week. :)

### Composition

I prefer the composition I chose as it's more balanced.

It's "bevy", so (to me) they're not shadows but actual individual birds. But even if they were shadows they wouldn't be rotated, but stay aligned to the object casting the shadows. So either way they should share a horizontal alignment axis, I think. It gives them a common direction of flight and a feel of perspective depth, when combined with perspective scaling/shading. Directional alignment is also one of the three forces of flocking.

Also in the current logo it looks to me like the back-most bird is kinda falling from the sky. A rather bad image for software. The Swift logo has the same issue and I've heard many people criticize it for that. 

### Details

Totally agree with you on "less (details) is more" front. :)

**Comment by awulkan** [comment]
I personally agree with @tamasfe about the pointy wings, it just looks more natural to me. The shape also looks a lot more like a bird. I don't think the birds have to be placed at an angle though. It looks more natural to place them in a somewhat straight line.

I also like the design @kettle11 did, but I'd lose the speed stripes and reduce the line thickness for the wing a bit.

Anyway, just trying to provide some feedback. Take it or leave it, it's your choice.

**Comment by GabCampbell** [comment]
I'm also on board with pointy wings and tail and after looking at the swift logo on a very small scale my opinion is that downscaling doesn't have enough of a significant impact to justify not using pointy wings and tail for that reason.

**Comment by GrantMoyer** [comment]
Here's my take on a very geometric adaption of the Bevy logo.

![bevy](https://user-images.githubusercontent.com/6333127/90968259-4cb0ec00-e4b8-11ea-8536-b99084a703cf.png)

[SVG files](https://github.com/bevyengine/bevy/files/5113329/geometric_bevy_logo.zip)

Although, I have to say, I'm pretty fond of tamasfe's more cartoonish adaption.

**Comment by rparrett** [comment]
I think @regexident has come close to nailing it.

I wanted to try something similar, with a few differences

- Original logo's head and beak shape
- Sharp wing tips, curved rear ring
- Round tail (mistake?)
- Each of the 3 birds "proudly holding their head up high"
- Happy eyes

What I ultimately produced is garbage due to being incompetent at inkscape (third bird's head is mangled, chest too pointy), but I think the eyes may be an improvement at least. Less of a death-stare feel. They may be a problem at smaller sizes though. I was unable to convince inkscape to enlarge and rotate them slightly ccw.

![bevy-circle](https://user-images.githubusercontent.com/200550/105220671-4a10fc00-5b15-11eb-9f48-530aae8af55b.png)

**Comment by awulkan** [comment]
@rparrett I like that one, great job. It looks much better when all birds are facing up, unlike the current logo where the last bird is facing down.

**Comment by rparrett** [comment]
In tweaking my version a bit further, I've really come to appreciate @regexident 's design must have evolved.

The overall proportions, head shape, and simplified geometry make sense when laying the birds out horizontally and making the logo legible at small sizes.

It's a great logo as-is. The only thing I'm not crazy about is that from front-to-back, if I attempt to assign emotions to the birds, I see:

- Proud
- Indifferent
- Bored

In my version, I rotated the body of the bird but kept the heads/beaks oriented the same way, and the third bird ends up with a  sort of determined / soaring look.

I think that tweaking the eyes alone might help with that too.

**Comment by alice-i-cecile** [comment]
@cart are we okay to close this one out? I think that the logo is fine / stable now, and we can re-open or create a new issue or discussion as the need arises.

**Comment by cart** [comment]
I'd actually prefer to keep this open. I am still very happy with the current logo, but I'm also happy to provide an outlet for people to express themselves. Every so often someone wants to take a whack at it (there have been some semi-recent iterations on discord and twitter). And theres always a chance that someone will propose something that should clearly be "the next bevy logo".

**Comment by cart** [comment]
(and a consolidated thread helps us keep this type of exploration centralized / easily scannable by interested people).

**Comment by konstantinkopka** [comment]
Hi! I made some really rough WIP logo sketches back in January that I wanted to share.

![IVersion A](https://cdn.discordapp.com/attachments/745355529777315850/881575436520394762/unknown.png)

![IVersion B](https://cdn.discordapp.com/attachments/745355529777315850/881575485539233822/unknown.png)

**Comment by alice-i-cecile** [comment]
Closing out: we have merchandise now so any rebranding would be a very significant effort.

**Comment by cart** [comment]
Opening because I still like this thread and want people to have an outlet for logo iteration.

**Comment by aevyrie** [comment]
I know this diverges from the feel of the original logo, but here's a direction I was exploring a while back - trying to make the logo scalable to small sizes with a more distinct silhouette. Not thrilled with the end result, but someone might find it useful as inspiration.

![image](https://user-images.githubusercontent.com/2632925/179316102-0312ab7f-4963-4130-a94d-180549455f6b.png)
![image](https://user-images.githubusercontent.com/2632925/179316119-27194930-f4ea-4b6f-ab3d-9e77edfd6690.png)
![image](https://user-images.githubusercontent.com/2632925/179316130-c3e91f4f-280c-47c8-89ef-dfa7a9fa9bf9.png)

FWIW I've had people ask me if my bevy sweatshirt was the Twitter logo. :laughing:

**Comment by UX3D-joshi** [comment]
Hello, I have an unrelated question. Can the current Bevy logo and name (using a similar font) form branding be used in company presentations or on websites? Is it licensed under a [Creative Commons Attribution-NoDerivatives 4.0 International License](http://creativecommons.org/licenses/by-nd/4.0/) ? thank you

**Comment by wild-fruits** [comment]
We need a mascot as a logo. It is good for advertising, something that can express more

**Comment by gregcsokas** [comment]
In this [issue](https://github.com/bevyengine/bevy/issues/15584) you can read more about the design process and idea. As @cart mentioned that's not the perfect place for a totally new design, so I move the last two comments content's here. It's still not finished yet. 

512x512
<img width="512" height="512" alt="bevy_logo_512" src="https://github.com/user-attachments/assets/d1140a90-d0a1-4224-b771-55d7b6a420cc" />

256x256
<img width="256" height="256" alt="bevy_logo_256" src="https://github.com/user-attachments/assets/227556ed-07f7-4f38-881c-4b9b685003bd" />

256x256 with "gaps" pointy
<img width="256" height="256" alt="bevy_logo_256_gaps_pointy" src="https://github.com/user-attachments/assets/9705a796-4784-4b0e-87b4-57b1a1a4059f" />

256x256 with "gaps" rounded
<img width="256" height="256" alt="bevy_logo_256_gaps_rounded" src="https://github.com/user-attachments/assets/63bfd21c-a94c-44df-9b34-6e47dbdfb0c1" />

128x128
<img width="128" height="128" alt="bevy_logo_128" src="https://github.com/user-attachments/assets/afa2fc9e-0714-4ef5-8337-d6ce72997dcb" />

64x64
<img width="64" height="64" alt="bevy_logo_64" src="https://github.com/user-attachments/assets/ce8b4e8a-0247-40d4-b93a-af13fa3a1db4" />

32x32
<img width="32" height="32" alt="bevy_logo_32" src="https://github.com/user-attachments/assets/ee4ebdc6-8436-4f26-8d6d-6450f6794fdc" />
